test/utils/dm/findDMForUser-test.ts

```diff
@@ -17,56 +17,112 @@ limitations under the License.
 import { mocked } from "jest-mock";
 import { MatrixClient, Room } from "matrix-js-sdk/src/matrix";
 
-import { DirectoryMember, ThreepidMember } from "../../../src/utils/direct-messages";
-import { MatrixClientPeg } from "../../../src/MatrixClientPeg";
 import DMRoomMap from "../../../src/utils/DMRoomMap";
-import { createTestClient } from "../../test-utils";
-import { findDMRoom } from "../../../src/utils/dm/findDMRoom";
+import { createTestClient, makeMembershipEvent } from "../../test-utils";
+import { LocalRoom } from "../../../src/models/LocalRoom";
 import { findDMForUser } from "../../../src/utils/dm/findDMForUser";
+import { getFunctionalMembers } from "../../../src/utils/room/getFunctionalMembers";
 
-jest.mock("../../../src/utils/dm/findDMForUser", () => ({
-    findDMForUser: jest.fn(),
+jest.mock("../../../src/utils/room/getFunctionalMembers", () => ({
+    getFunctionalMembers: jest.fn(),
 }));
 
-describe("findDMRoom", () => {
+describe("findDMForUser", () => {
```

Comment:
This file contained the wrong test :laughing:
While splitting up the „start DM only on first message“ PR I seem to have mixed it up.